content/en/users/aai/check-in/faq/_index.md

```diff
@@ -31,15 +31,15 @@ credentials from a Dutch IdP in eduGAIN, you or your IdP administrators need to
 request the connection. The following steps will lead you to perform the
 connection:
 
-- Connect to [SURFconext dashboard](https://dashboard.surfconext.nl/apps)
-- Search for "EGI AAI Service provider proxy"
+- Connect to [SURFconext dashboard](https://dashboard.surfconext.nl/apps/all)
+- Search for "EGI Check-in Service"
```

Comment:
I would suggest something like:
```
- Open the [SURFconext dashboard](https://dashboard.surfconext.nl/apps/all) and log in
```

Reply:
`you or your IdP administrators need to request the connection` to be replaced by something like `the SURFConext administrator need to request the connection`. It's likely the only one able to do this.

Reply:
Thanks!